python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -323,17 +288,6 @@ def trackandRelease(self):
 
                  flush_thread = threading.Thread(target=self._flush, args=())
                  flush_thread.run()
-
-                 for obj in self.dirty_objs.itervalues():
-
-                    if self.shouldReleaseRun is False:
-                        break
-
-                    _args = ([obj])
-                    release_thread = threading.Thread(target=self._release_lock, args=_args)
```

Comment:
I am fairly sure that we don't allow submission from multiple Ganga sessions anyway (at least to the best of my knowledge). You can have multiple Ganga sessions but only one has write access. Isn't this why we have the session files created?